django_declarative_apis/machinery/tasks.py

```diff
@@ -276,6 +286,145 @@ def schedule_future_task_runner(
                     raise err
 
 
+@celery_task(
+    ignore_results=True,
+    time_limit=getattr(settings, "DDA_DEFERRED_TASK_TIME_LIMIT", 999999),
+    soft_time_limit=getattr(settings, "DDA_DEFERRED_TASK_SOFT_TIME_LIMIT", 999999),
+)
+def generic_future_task_runner(
+    endpoint_class_name,
+    endpoint_method_name,
+    packed_args,
+    packer_class_name,
+    task_job_count=0,
+    task_creation_time=None,
+    scheduled_execution_delay=0,
+    retry_params=None,
+    correlation_id=None,
+):
+    endpoint_class = locate(endpoint_class_name)
+    endpoint_task = getattr(endpoint_class, endpoint_method_name)
+    packer_class = locate(packer_class_name)
+    args, kwargs = packer_class.unpack(packed_args)
+
+    _set_correlation_id(correlation_id)
+
+    _log_task_stats(
+        endpoint_method_name,
+        None,
+        scheduled_execution_delay,
+        task_creation_time,
+        task_job_count,
+        correlation_id,
+    )
+
+    logger.info(
+        "future_task_runner: method=%s, resource_id=%s",
+        endpoint_method_name,
+        None,
+    )
+
+    try:
+        endpoint_task.task_runner(*args, **kwargs)
+    except Exception as e:
+        if retry_params is None:
+            raise
+
+        retry_params = RetryParams(*retry_params)
+        if retry_params.retry_exception_filter and not any(
+            [
```

Comment:
You could probably drop the square brackets and let it be a generator expression?